types/node/events.d.ts

```diff
@@ -214,9 +219,15 @@ declare module "events" {
          * ```
          * @since v11.13.0, v10.16.0
          */
-        static once(
-            emitter: NodeJS.EventEmitter,
-            eventName: string | symbol,
+
+        static once<T extends EventMap<T> = DefaultEventMap, K extends EventMapKey<T> = any>(
```

Comment:
Are defaults for `K` needed here? My instinct is no, but if there is a case for them, then they probably shouldn't be `any`.